packages/graphql/tests/integration/issues/283.int.test.ts

```diff
@@ -47,7 +47,7 @@ describe("https://github.com/neo4j/graphql/issues/283", () => {
         type Post {
             id: ID! @id @unique
             title: String!
-            datetime: DateTime @readonly @timestamp(operations: [CREATE])
+            datetime: DateTime @timestamp(operations: [CREATE])
```

Comment:
Should this not make use of @settable so it uses the same schema?

```suggestion
            datetime: DateTime @timestamp(operations: [CREATE]) @settable(onCreate: false, onUpdate: false)
```